src/pages/index.tsx

```diff
@@ -10,16 +13,44 @@ import {
 import { type NextPage } from 'next';
 import AddCharacterForm from '../components/AddCharacterForm';
 
+import SearchBar from '../components/SearchBar';
 import { api } from '../utils/api';
 
 const Home: NextPage = () => {
   const characters = api.characterRouter.getCharacters.useQuery()?.data;
 
   return (
     <Center minH="100vh" backgroundColor="gray.900">
-      <VStack spacing={5}>
+      <Image
+        top={0}
+        linear-gradient="(135deg, #50A68480 30%, #115E6780 90%)"
```

Comment:
This is super cool!

Reply:
pulled straight from the figma hah not my work :)